mlflow/store/sqlalchemy_store.py

```diff
@@ -118,12 +119,13 @@ def create_run(self, experiment_id, user_id, run_name, source_type, source_name,
         if experiment.lifecycle_stage != LifecycleStage.ACTIVE:
             raise MlflowException('Experiment id={} must be active'.format(experiment_id),
                                   INVALID_STATE)
-        status = RunStatus.to_string(RunStatus.RUNNING)
```

Comment:
Note: create_experiment should return experiment id; `return experiment.experiment_id`